deepchem/utils/test/test_dftutils.py

```diff
@@ -0,0 +1,32 @@
+"""
+Test for DFT Utilities
+"""
+import dqc
+from dqc.system.mol import Mol
+from dqc.qccalc.ks import KS
+from deepchem.utils.dftutils import KSCalc, hashstr
+import torch
+
+
```

Comment:
@pytest.mark.torch because dqc requires pytorch to be installed

Reply:
Since we are running dqc tests in a separate workflow, we should have it marked as @pytest.mark.dqc and then invoke the test in dqc workflow.